Formula/abcl.rb

```diff
@@ -17,15 +17,14 @@ class Abcl < Formula
   depends_on "rlwrap"
 
   def install
-    cmd = Language::Java.java_home_cmd("1.8")
-    ENV["JAVA_HOME"] = Utils.safe_popen_read(cmd).chomp
+    ENV["JAVA_HOME"] = Language::Java.java_home("1.8")
 
     system "ant"
 
     libexec.install "dist/abcl.jar", "dist/abcl-contrib.jar"
     (bin/"abcl").write <<~EOS
       #!/bin/sh
-      export JAVA_HOME=$(#{cmd})
+      export JAVA_HOME=#{ENV["JAVA_HOME"]}
```

Comment:
Ideally this would actually use `Language::Java::overridable_java_home_env` so it doesn't just take the path where the JDK is installed on the CI.

Reply:
Could you demonstrate how to use that in this context? I tried just replacing the entire `ENV["JAVA_HOME"]` with it and evidently that's not how this is supposed to be used.

Reply:
Not tested this but:

If possible, use `write_env_script`. So something like:
```rb
(bin/"abcl").write_env_script "rlwrap" , "java -cp #{libexec}/abcl.jar:\"$CLASSPATH\" org.armedbear.lisp.Main \"$@\"", Language::Java::overridable_java_home_env("1.8")
```

Otherwise using `Language::Java::overridable_java_home_env("1.8")[:JAVA_HOME]` (or perhaps a loop is better) is possible if `write_env_script` proves impossible.